docs/use-linea/explore/use-match3-game.md

```diff
@@ -0,0 +1,67 @@
+- With intense PvP and PvE battles, you'll have the opportunity to become a legend in the arena. Join open tournaments for a chance to win prizes.
+- Capture, collect, and merge monsters to create even stronger and more unique creatures. With endless possibilities and combinations, the sky's the limit in creating the ultimate team.
+
+Meta Merge Match3 is a mini game that allows everyone to quickly get to know pets and achieve Linea network accomplishments.
+
+## What is Meta Merge Match3 Game?
+
+🕹️ Meta Merge Match3 is an elimination game developed by Meta Merge Team. In the game, players need to select three identical pets to eliminate them with some skill😉.
+
+- The pets will continue to overlap, and players can only select the top layer of them.
+- There could be more than 30 types of pets in the game
+- Players can obtain by Ultiverse account or wallet connection.
+- The game may become very difficult and challenging after the high level.
+
+![](../../assets/meta_merge/match3_intro.png)
+
+I hope this information helps! If you have any other questions, feel free to ask me.
+
+## To Start:
+
+### Visit Meta Merge Match3 and Login.
+
+Visit the Meta Merge official website: https://match3-linea.metamerge.xyz/ and youwill find that there are 2 ways of logging in.
+
+1. If you already have an Ultiverse account, you may login directly with your Ultiverse Account. You can also sign up your Ultiverse account here.
+2. (recommended) Login with your preferred wallet address. You will need to connect and sign with your selected digital wallet.
+
+![](../../assets/meta_merge/account_login.png)
+
+### Confirm wallet and start the Game
+
+Once you have successfully logged in, you can click on the "Linea Game" link at the top to enter the mini-game. Before starting the game, please select the connected wallet with which you will interact during the game. Only then can you begin playing. If you logged with account, You can choose any of the wallets under your account.
+
+![](../../assets/meta_merge/match3_select_wallet.png)
+
+After players click on the cards in the central area of the game, the cards will move up to the top frame. The top frame can store a total of 7 cards. When there are 3 identical cards, they will be eliminated, for example, three of the same cards will result in an elimination. Then click "Play" and let's get started!
+
+![](../../assets/meta_merge/match3_game_screenshot.png)
+
+When there are no cards that can be eliminated by forming a set of 3 on the game board, players can observe the temporarily unused cards in the lower layer. By clicking on the cards in the upper layer, the cards in the lower layer become available for use. However, when using this technique, it is important to avoid accumulating useless cards in hand, as it can ultimately lead to game failure.
+
+If the cards in hand has already been 7 and there are not 3 indentical cards here. Then unfortunately
```

Comment:
<img width="1096" alt="Screenshot 2023-06-13 at 10 19 54 AM" src="https://github.com/ConsenSys/doc.zk-evm/assets/17280653/d1498dc0-57d0-4168-8170-7f825defe27b">
Can you fix all the spelling errors?

If it needs to be a real word, you can:
- allowlist on `project-words.txt`
- run `npm run lint` to check if it's correct
- fix, squash commits, and force push